flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -1185,8 +1295,10 @@ public DataStreamSource<Long> fromSequence(long from, long to) {
      * @param data The array of elements to create the data stream from.
      * @param <OUT> The type of the returned data stream
      * @return The data stream representing the given array of elements
+     * @deprecated Use {@link #fromData(OUT...)} instead
```

Comment:
Maybe be explicit about a scheduled removal in 2.0

Reply:
Sure, but the formulation has to tentative. While this is still my goal, you probably remember this long thread were the item was dropped as a must-have for 2.0 : https://lists.apache.org/thread/r0y9syc6k5nmcxvnd0hj33htdpdj9k6m

How about this:
```
This method will be removed a future release, possibly as early as version 2.0.  Use {@link #fromData(OUT...)} instead.
```

Reply:
eh fair enough, Ill merge it as is. 🤷